pages/common/ansible.md

```diff
@@ -0,0 +1,27 @@
+# ansible
+
+> Manage groups of computers remotely over SSH
+
+- Display a list of hosts belonging to a group:
+
+`ansible {{group}} --list-hosts`
+
+- Ping a group of hosts:
+
+`ansible {{group}} --module-name ping`
+
+- Display facts about a group of hosts:
+
+`ansible {{group}} --module-name setup`
```

Comment:
> By using single-letter options, it will conflict with contribution guideline point 3.

You may be misreading the guideline. It says "Try to incorporate the spelled-out version of single-letter options **in the example's description**" (emphasis mine). If we can't add them to the description, expanding them in the command itself it an acceptable alternative, but it may even be that just the option's value can make it clear (e.g. in `-i {{path/to/inventory/file}}`, the token name allows one to infer that `-i` stands for "inventory". At the end of the day, this whole process is rather subjective, which is why there are guidelines, not hard-and-fast rules.

As for the phrasing, I agree that the second option is more helpful. As long as the relevant information is preserved, I see no problems putting the "what" first and the "how" later.